CONTENTdm.js

```diff
@@ -0,0 +1,577 @@
+		legacyScrapeToItem( doc, item, headerXPath, contentXPath, offset, legacyMapping, docTitle, url );
+	  }
+
+	  else if ( fullSource.match( "CONTENTdm" ) ) { // Web scrape for CDM Version 4 or below
+		headerXPath = '//div/div[3]/table[2]/tbody/tr/td[1]';
+		contentXPath = '//div/div[3]/table[2]/tbody/tr/td[2]';
+		offset = 0;
+		legacyScrapeToItem( doc, item, headerXPath, contentXPath, offset, legacyMapping, docTitle, url );
+	  }
+
+	  item.complete();
+
+	  return; // Return from the ZU.doGet callback after Web scrape
+	}
+  } );
+  return '';
+}
+
+function legacyScrapeToItem( doc, item, headerXPath, contentXPath, offset, mapping, docTitle, url ) {
+  var headers = ZU.xpath( doc, headerXPath ); // Scrapes the whole metadata table and looks for matches after.
+  var contents = ZU.xpath( doc, contentXPath );
+  for ( i = 0; i < contents.length; i++ ) {
+	var header;
+	var content;
+	j = i + offset;
+	if ( headers[ j ].textContent ) {
+	  header = headers[ j ].textContent;
+	  header = camelize( header );
+	}
+	if ( contents[ i ].textContent ) content = contents[ i ].textContent;
+	item[ header ] = content;
+  }
+  for ( var i in mapping ) item[ mapping[ i ][ 0 ] ] = item[ mapping[ i ][ 1 ] ];
+  tidyItem( item );
+  if ( !item.title ) item.title = getTitle( doc, docTitle, url );
+
+  function camelize( str ) {
+	return str.replace( /(?:^\w|[A-Z]|\b\w)/g, function( letter, index ) {
+	  return index === 0 ? letter.toLowerCase() : letter.toUpperCase();
+	} ).replace( /\s+/g, '' );
+  }
+}
+
+function scrapeToItem( doc, item, url, docTitle, mapping, prefixes, doc2 ) {
+  var info = []; // Scrapes only fields named in the mapping array.
+  var xpath;
+  var itemText;
+  for ( var i in mapping ) {
+	var thisField = [];
+	var field = mapping[ i ][ 1 ];
+	if ( prefixes == "web" ) xpath = {
+	  object: "//*[@id='metadata_object_" + field + "']",
+	  item: "//*[@id='metadata_" + field + "']"
+	};
+	else if ( prefixes == "xml" ) xpath = {
+	  object: "//" + field,
+	  item: ""
+	};
+	thisField = [ mapping[ i ][ 0 ], xpath.object, xpath.item ];
+	info.push( thisField );
+  }
+  for ( var j in info ) {
+	var text = fetchXPathText( doc, info[ j ][ 1 ] );
+	if ( !doc2 ) { // Scrape from single document
+	  if ( info[ j ][ 0 ] == "title" && !text ) text = getTitle( doc, docTitle, url );
+	  if ( info[ j ][ 2 ] ) itemText = fetchXPathText( doc, info[ j ][ 2 ] );
+	} else { // Scrape from two documents
+	  itemText = fetchXPathText( doc2, info[ j ][ 1 ] );
+	  if ( info[ j ][ 0 ] == "title" && !itemText ) itemText = getTitle( doc2, docTitle, url );
+	}
+	if ( text != itemText ) text = [ itemText, text ].filter( function( val ) {
+	  return val;
+	} ).join( '. ' );
+	item[ info[ j ][ 0 ] ] = text;
+  }
+  tidyItem( item );
+  return item;
+}
+
+function tidyItem( item ) {
+  mergeFields( item );
+  replaceFields( item );
+  cleanAuthor( item );
+  item.tags = []; //Tags must be in an array
+  item.tags = item.subject.replace( /\([^)]*\)/g, '' ).split( /[.;]/ );
```

Comment:
I'd check for item.subject here to be safe -- otherwise `.replace()` will trigger an error when it's empty.